x-pack/plugins/actions/server/usage/actions_telemetry.ts

```diff
@@ -119,14 +125,17 @@ export async function getInUseTotalCount(
   countEmailByService: Record<string, number>;
   countNamespaces: number;
 }> {
-  const scriptedMetric = {
+  const getInMemoryActionScriptedMetric = (actionRefPrefix: string) => ({
```

Comment:
I switch the order of `scriptedMetric` and `preconfiguredActionsScriptedMetric` (which I made it a function)